lib/core/address_validator.dart

```diff
@@ -53,6 +61,7 @@ class AddressValidator extends TextValidator {
               '|(bc1q[ac-hj-np-z02-9]{25,39})'
               '|(bc1p([ac-hj-np-z02-9]{39}|[ac-hj-np-z02-9]{59}|[ac-hj-np-z02-9]{8,89}))'
               '|(bc1q[ac-hj-np-z02-9]{40,80})'
+              '|(lightning:)?(lnbc|lntb|lnbs|lnbcrt|lnurl)[a-z0-9]'
```

Comment:
Also, are you sure it's gonna be always lower case letters?
```suggestion
              '|(lightning:)?(lnbc|lntb|lnbs|lnbcrt|lnurl)[a-z0-9]+'
```